qemu/tests/virtio_fs_set_capability.py

```diff
@@ -0,0 +1,304 @@
+        check_virtiofs_cmd = params["check_virtiofs_cmd"]
+
+        if not utils_misc.wait_for(lambda: re.search("virtiofs",
+                                                     session.cmd_output(
+                                                         check_virtiofs_cmd),
+                                                     re.IGNORECASE), 30):
+            test.fail("Virtiofs service is failed to start.")
+
+        virtio_fs_disk_label = fs_target
+        error_context.context("Get Volume letter of virtio fs target, the disk"
+                              "lable is %s." % virtio_fs_disk_label,
+                              logging.info)
+        vol_con = "VolumeName='%s'" % virtio_fs_disk_label
+        vol_func = utils_misc.get_win_disk_vol(session, condition=vol_con)
+        volume_letter = utils_misc.wait_for(lambda: vol_func, 120)
+        fs_dest = "%s:" % volume_letter
+
+    guest_file = os.path.join(fs_dest, 'fs_test')
+    logging.info("The guest file in shared dir is %s.", guest_file)
+
+    try:
+        # No extended attributes (file steams) in virtio-fs for windows
+        if not is_windows:
+            if cmd_set_trusted:
+                host_attributes = params["host_attributes"]
+                guest_trusted = params["guest_trusted"]
+                file_capability = params["file_capability"]
+                error_context.context("Set a trusted on guest.", logging.info)
+                session.cmd(cmd_yum_attr)
+                session.cmd(cmd_set_trusted)
+                output = session.cmd_output(cmd_get_trusted)
+                logging.info("The guest file trusted is %s.", output)
+                if guest_trusted not in output:
+                    test.fail("It's failed to check the trusted info from the guest.")
+
+                process.run(cmd_yum_attr)
+                output = str(process.run('getfattr %s' % fs_source).stdout.strip())
+                logging.info("The host file trusted is %s.", output)
+                if host_attributes not in output:
+                    test.fail("It's failed to check the trusted info from the host.")
+
+                session.cmd(cmd_create_file)
+                logging.info("Set capability for guest files.")
+                session.cmd(cmd_set_capability)
+                output = session.cmd_output(cmd_get_capability)
+                logging.info("The guest file capability is %s.", output)
+                if file_capability not in output:
+                    test.fail("It's failed to check the capability of the guest.")
+                logging.info("Modify file content and check the file capability.")
+                session.cmd(cmd_echo_file)
+                output = session.cmd_output(cmd_get_capability)
+                logging.info("The guest change file capability is %s.", output)
+                if file_capability in output:
+                    test.fail('Capability result doesn‘t meet expectations.')
```

Comment:
test.fail('Still can get capability after file content is changed.')

Reply:
Corrected